aip/0180.md

```diff
@@ -0,0 +1,126 @@
+---
+aip:
+  id: 180
+  state: reviewing
+  created: 2019-07-23
+permalink: /180
+redirect_from:
+  - /0180
+---
+
+# Backwards compatibility
+
+APIs are fundamentally contracts with users, and users often write code against
+APIs that is then launched into a production service with the expectation that
+it continues to work (unless the API has a [stability level][aip-181] that
+indicates otherwise). Therefore, it is important to understand what constitutes
+a backwards compatible change and what constitutes a backwards incompatible
+change.
+
+## Guidance
+
+Existing client code **must not** be broken by a service updating to a new
+minor or patch release. Old clients **must** be able to work against newer
+servers (with the same major version number).
+
+**Important:** It is not always salient whether a change is compatible or not.
```

Comment:
I'm not sure what you mean by "salient" here, and I suspect people without English as a primary language will find it even more confusing. Let's try to lower the required reading age in general :)

Reply:
Fixed.